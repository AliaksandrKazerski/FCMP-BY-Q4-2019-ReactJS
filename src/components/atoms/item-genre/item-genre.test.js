import React from 'react';
import { shallow } from 'enzyme';

import ItemGenre from './index';

describe('ItemGenre', () => {
  it("ItemGenre should render correctly", () => {
  const myComponent = shallow(<ItemGenre/>);
  
  expect(myComponent).toMatchSnapshot()
  });
});