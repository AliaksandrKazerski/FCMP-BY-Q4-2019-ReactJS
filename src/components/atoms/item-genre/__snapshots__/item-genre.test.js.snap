// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemGenre ItemGenre should render correctly 1`] = `ShallowWrapper {}`;
